packages/webapp/pages/squads/[handle]/edit.tsx

```diff
@@ -90,14 +87,7 @@ const EditSquad = ({ handle }: EditSquadPageProps): ReactElement => {
   return (
     <ManageSquadPageContainer>
       <NextSeo {...seo} titleTemplate="%s | daily.dev" noindex nofollow />
-      <ManageSquadPageMain>
```

Comment:
This not needed anymore?

Reply:
Yep. This is not needed anymore on both files (due to some recent changes), hence, the classes were applied on the container at the `Details` component.